README.md

```diff
@@ -227,166 +215,171 @@ const sid1 = nc.subscribe(subj, {queue: 'q1'}, () => {
 // messages and drained connection is closed.
 // - finally, the callback handler is called (with possibly an error).
 
-let c2 = 0;
-nc.subscribe(subj, {queue: 'q1'}, () => {
-    c2++;
-    if(c2 === 1) {
-        nc1.drain(() => {
-            // connection drained - possible arguments is an error
-            // connection is closed by the time this function is
-            // called.
-        });
-    }
-});
+let c2 = 0
+nc.subscribe('foo', { queue: 'q1' }, () => {
+  c2++
+  if (c2 === 1) {
+    nc.drain(() => {
+      // connection drained - possible arguments is an error
+      // connection is closed by the time this function is
+      // called.
+    })
+  }
+})
+
 ```
 
 ## TLS
 
 ```javascript
-var nats = require('nats');
-var fs = require('fs');
+const NATS = require('nats')
+const fs = require('fs')
 
 // Simple TLS connect
-var nc = nats.connect({port: TLSPORT, tls: true});
+let nc = NATS.connect({ tls: true })
```

Comment:
Do we need this? If server requires TLS it should just do the right thing yes?

Reply:
this is saying that the client requires the connection to be tls.